pkg/sharing/sharing.go

```diff
@@ -196,6 +197,12 @@ func (s *Sharing) CreateRequest(inst *instance.Instance) error {
 	s.UpdatedAt = time.Now()
 	s.Credentials = make([]Credentials, 1)
 
+	for i, m := range s.Members {
+		if contact, err := contacts.FindByEmail(inst, m.Email); err == nil {
```

Comment:
`err != nil` ?

Reply:
No, it's if `err` is nil that the contact is found and valid and that we can use its name. We can safely ignore errors here.

Reply:
My mistake, I should have reviewed this review